coops-ui/src/mixins/common-mixin.ts

```diff
@@ -1,18 +1,47 @@
 import { Component, Vue } from 'vue-property-decorator'
 import omit from 'lodash.omit'
+import isEqual from 'lodash.isequal'
 
 /**
  * Mixin that provides some useful common utilities.
  */
 @Component({})
 export default class CommonMixin extends Vue {
   /**
-   * Removes the specified property from an object
+   * Removes the specified property from an object.
    *
-   * @param baseObj The base object
-   * @param prop The property to be removed
+   * @param baseObj The base object.
+   * @param prop The property to be removed.
    */
   omitProp (baseObj: Object, prop: Array<string>): Object {
```

Comment:
I'm thinking this should be `object` instead of `Object`. What do you think?

Reply:
I think when defining a type it is proper to go with `Object` because it is an actual Javascript Type, wheres `object` is an instance of `Object` and could really be anything that holds something.

Reply:
I'm not convinced. It's kind of like string vs String (though I understand that one better). IMO baseObj is a generic 'object' while Object should be used to get its prototypes (methods), which is not what we're doing here. I'll read up on it a bit more. Both probably work in the code.

Reply:
i'll change it.

Reply:
Done, thanks.